### DISCUSSION TITLE
Is there any grafana dashboard for visualize kong/kubernetes-ingress-controller?

### DISCUSSION THREAD

**Discussion by ssam13** [Kong/kubernetes-ingress-controller#1974] [Q&A] [answered]
Hi everybody 👋

I've installed **Prometheus** and **Grafana** on my **Kubernetes cluster** using `prometheus-community/kube-prometheus-stack` helm charts.
I've also installed **Prometheus-Plugin** of `kong/kubernetes-ingress-controller` and defined a **serviceMonitor** to scrape metrics of my **ingress-controller**.
Now I can see **kong's metrics** in **Prometheus UI** as a **target**.

The question is how can I visualize kong-ingress metrics in grafana?!

There is a dashboard in grafana with id **7424** for kong (**NOT** kong-ingress), that can not read metrics from kong-ingress. 

Are there any other choices?
 
Thanks 🙏

**Comment by shaneutt** [comment]
At the time of writing there isn't one, but we are actively working on this for an upcoming patch release: https://github.com/Kong/kubernetes-ingress-controller/issues/1926

**Comment by rainest** [comment, marked as answer]
There is now a Grafana dashboard available: https://github.com/Kong/kubernetes-ingress-controller/blob/main/grafana.json
